owsi-core/owsi-core-components/owsi-core-component-jpa/src/main/java/fr/openwide/core/jpa/batch/executor/SimpleHibernateBatch.java

```diff
@@ -0,0 +1,128 @@
+package fr.openwide.core.jpa.batch.executor;
+
+import java.util.Collection;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.TransactionStatus;
+import org.springframework.transaction.support.TransactionCallback;
+
+import com.google.common.collect.Lists;
+import com.querydsl.core.types.dsl.PathBuilder;
+import com.querydsl.jpa.impl.JPAQuery;
+
+import fr.openwide.core.commons.util.functional.Joiners;
+import fr.openwide.core.jpa.business.generic.model.GenericEntity;
+import fr.openwide.core.jpa.business.generic.model.QGenericEntity;
+import fr.openwide.core.jpa.exception.ServiceException;
+
+@Component
+@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+public class SimpleHibernateBatch extends AbstractBatch<SimpleHibernateBatch> {
```

Comment:
Since the additional features of this class are disabled by default, maybe we could directly add these features to the MultithreadedBatchExecutor?